src/wxchartoptions.cpp

```diff
@@ -51,3 +52,15 @@ bool wxChartOptions::IsSaveAsMenuEnabled() const
 {
     return m_enableSaveAsMenu;
 }
+
+void wxChartOptions::SetSaveAsMenuEnabled(bool enable) {
```

Comment:
Opening curly brace `{` on the same line doesn't conform to the coding style in this file. Not a big issue though :)